proto/ibc/applications/interchain_accounts/v1/genesis.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+package ibc.applications.interchain_accounts.v1;
+
+option go_package = "github.com/cosmos/ibc-go/modules/apps/27-interchain-accounts/types";
+
+import "gogoproto/gogo.proto";
+
+// GenesisState defines the interchain_account genesis state
+message GenesisState {
+  string port_id = 1 [(gogoproto.moretags) = "yaml:\"port_id\""];
```

Comment:
@colin-axner I am not entirely sure what we want to put in our genesis state. :thinking:

Reply:
Any state that needs to be persisted between dump state/restart upgrades